static/app/utils/groupCapabilities.tsx

```diff
@@ -0,0 +1,29 @@
+import {BaseGroup, IssueCategory, IssueCategoryCapabilities} from 'sentry/types/group';
+
+/**
+ * Defines what capabilities each category of issue supports
+ */
+const ISSUE_CATEGORY_CAPABILITIES: Record<IssueCategory, IssueCategoryCapabilities> = {
+  [IssueCategory.ERROR]: {
+    delete: true,
+    deleteAndDiscard: true,
+    ignore: true,
+    merge: true,
+  },
+  [IssueCategory.PERFORMANCE]: {
+    delete: false,
+    deleteAndDiscard: false,
+    ignore: false,
+    merge: false,
+  },
+};
+
+/**
+ * Checks if an issue supports a specific capability.
+ */
+export function issueSupports(
+  issue: BaseGroup,
```

Comment:
nitpick but I think I'd rather pass just `issueCategory` here so the consumer doesn't need the whole issue object

Reply:
yeah I think I agree. Updated